nifi-nar-bundles/nifi-ranger-bundle/nifi-ranger-plugin/src/test/java/org/apache/nifi/ranger/authorization/TestRangerNiFiAuthorizer.java

```diff
@@ -160,7 +156,7 @@ public void testKerberosEnabledWithoutPrincipal() {
 
         try {
             authorizer.onConfigured(configurationContext);
-            Assert.fail("Should have thrown exception");
+            fail("Should have thrown exception");
```

Comment:
This `fail()` call should be refactored to use `assertThrows()`.

Reply:
FYSA, I lost a lot of these when I cherry-picked the base commits into new, smaller branches.

Reply:
Thanks for background.  Although this approach is more incremental, these smaller pull requests have been easier to review, so thanks for keeping the upgrade process moving!